docs/README.md

```
@@ -45,6 +54,17 @@ WandererKills provides:
 - Multiple integration patterns for different use cases
 - Comprehensive monitoring and observability
 
+### Simplified Processing Pipeline
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)